assets/xml/objects/object_icefloe.xml

```diff
@@ -1,7 +1,7 @@
 ﻿<Root>
     <File Name="object_icefloe" Segment="6">
-        <DList Name="object_icefloe_DL_0001E0" Offset="0x1E0" />
-        <Texture Name="object_icefloe_Tex_0002C8" OutName="tex_0002C8" Format="rgba16" Width="32" Height="32" Offset="0x2C8" />
-        <Collision Name="object_icefloe_Colheader_000C90" Offset="0xC90" />
+        <DList Name="gIcePlatform_Dlist" Offset="0x1E0" />
```

Comment:
```suggestion
        <DList Name="gIcefloeIcePlatformDL" Offset="0x1E0" />
```
Nice work on documenting the object. 
Since these are automatically global, it's a good idea to prefix them with a unique identifier. In this case the name of the object works and no `_` is needed between the suffix (note both these suggestions apply to all three). For `DLists` in particular the prefix used is `DL`.